ddtrace/llmobs/_integrations/langchain.py

```diff
@@ -203,3 +203,50 @@ def record_usage(self, span: Span, usage: Dict[str, Any]) -> None:
         total_cost = span.get_metric(TOTAL_COST)
         if total_cost:
             self.metric(span, "incr", "tokens.total_cost", total_cost)
+
+    def tag_lcel_inputs(self, span: Span, inputs):
+        """
+        Tags the span with the inputs to the LCEL chain,
+        taking into account for the differences in structure (invocation, batching).
+        Also, puts these values into a dictionary for returning an object for logs.
+        """
+        if isinstance(inputs, list):
```

Comment:
Instead of tagging `langchain.request.input` vs `langchain.request.inputs.%d` and handling separate cases, we can simplify this by ensuring `inputs` is a list and always tagging `langchain.request.inputs.%d`. What do you think?

Reply:
i.e. something like
```python
if not isinstance(inputs, list):
    inputs = [inputs]
# do what you did for the list case
```

Reply:
good point, I think that's a fair approach